doc(scheduler_perf): enrich the documentation (#122764)
kubernetes/kubernetes: test/integration/scheduler_perf/README.md

# Scheduler Performance Test

> This package contains the scheduler performance test, often called scheduler_perf.

Reviewer: ```suggestion
This package contains the scheduler performance tests, often called scheduler_perf.
```

We use it for benchmarking the scheduler with in-tree plugins, which is visible at [perf-dash](https://perf-dash.k8s.io/#/?jobname=scheduler-perf-benchmark&metriccategoryname=Scheduler&metricname=BenchmarkPerfResults&Metric=SchedulingThroughput&Name=SchedulingBasic%2F5000Nodes%2Fnamespace-2&extension_point=not%20applicable&result=not%20applicable).

> And you can also use it outside of Kubernetes repository with out-of-tree plugins by make use of `RunBenchmarkPerfScheduling`.

Reviewer: ```suggestion
Also you can use it outside the Kubernetes repository with out-of-tree plugins by making use of `RunBenchmarkPerfScheduling`.
```
